rules/windows/builtin/win_alert_ad_user_backdoors.yml

```diff
@@ -19,6 +19,8 @@ logsource:
 detection:
     selection1:
         EventID: 4738
+    filter_null:
+        AllowedToDelegateTo: null
     filter1:
         AllowedToDelegateTo: 
             - null
```

Comment:
The filter that you try to add should already be active with the expression in line 26

Reply:
This is what I meant in #777. I think the ES backend has troubles converting the list entry into the correct not-null expression.